exercises/practice/high-scores/.meta/lib/example.dart

```diff
@@ -0,0 +1,46 @@
+class HighScores {
+  List<int> scores = [];
+
+  HighScores(List<int> inputScores) {
+    this.scores = inputScores;
+  }
+
+  List<int> get getScores {
```

Comment:
In Dart, [when you define a `getter`, you don't include `get` in the name.](https://dart.dev/guides/language/effective-dart/design#avoid-starting-a-method-name-with-get)

`getter`s are generally used  (and explicitly in this case) to allow private properties to be access without altering their values.